tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -534,7 +516,7 @@ describe('Home - Create Game', () => {
       .then((games) => {
         expect(games.length).to.eq(1, 'Expect exactly 1 game in store');
         expect(games[0].numPlayers).to.eq(0, 'Expect no players in gameLists game in store, but found some');
-        expect(games[0].status).to.eq(true, 'Expect game to have status true');
+        expect(games[0].status).to.eq(GameStatus.CREATED, 'Expect game to have status CREATED'); //status = created
```

Comment:
Comment is no longer needed
```suggestion
        expect(games[0].status).to.eq(GameStatus.CREATED, 'Expect game to have status CREATED');
```